stellargraph/data/explorer.py

```diff
@@ -953,3 +954,31 @@ def _walk(self, src, dst, t, length, bias_type, np_rs):
                 break
 
         return walk
+
+
+class ConcurrentGraphWalk:
+    """
+    Internal utility class for using GraphWalk classes in a multi-threaded
+    environment.
+
+    """
+
+    def __init__(self, create_with_seed, rs):
```

Comment:
Maybe instead of taking a `rs`, this could take a seed and manage its own randomness internally?

Reply:
makes sense 👍